app/services/create_order_service.rb

```diff
@@ -0,0 +1,61 @@
+class CreateOrderService
```

Comment:
It would be awesome if this class had some test coverage

Reply:
I agree, but acceptance criterias in this task do not mention about tests so I didn't know if I should go further :) Of course I can try implementing tests for that feature :)

Reply:
As before, up to you what you'd like to train. Just please let me know so I can approve the PR at your desired state.

Reply:
I will implement tests 👍 Thus please wait with approving the PR until the tests will be commited :)

Reply:
👍

Reply:
Thanks for your waiting :) I added tests for both service objects. Could you take a look at them?